pages/network/Share Private Network/guide.en-gb.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Network - How to share a private network between 2 public cloud projects
+excerpt: Learn how to share a private network between two public cloud projects on OVH Cloud
+updated: 2024-06-20
+---
+
+## Objective
+
+This guide will show you how to connect two projects on an internal network. It allows you to synchronize production and preproduction environments across several projects.
+
+## Capabilities and limitations
+
+### Prerequisites
+
+- Have 2 or more projects.
+
+### Procedure
```

Comment:
Wrong term (Instructions) and markdown hierarchy (use an H2 then adapt your hierarchy from it)